lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -140,55 +146,55 @@ python-neutronclient==7.2.1; python_version >= "3.6"
+pytz==2021.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6"
 pyuwsgi==2.0.19.1.post0
 pyyaml==5.4.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0")
 rdflib-jsonld==0.5.0; python_version >= "3.6" and python_version < "4"
 rdflib==4.2.2; python_version >= "3.6" and python_version < "4"
-refgenconf==0.9.3
+refgenconf==0.10.0
 repoze.lru==0.7
 requests-oauthlib==1.3.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0"
-requests-toolbelt==0.9.1; python_version >= "3.5"
+requests-toolbelt==0.9.1; python_version >= "3.6"
 requests==2.25.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0")
 requestsexceptions==1.4.0; python_version >= "3.6"
 rfc3986==1.4.0; python_version >= "3.6"
+rich==9.13.0; python_version >= "3.6" and python_version < "4.0"
 routes==2.5.1
-rsa==4.7; python_version >= "3.5" and python_version < "4" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
+rsa==4.7.2; python_version >= "3.5" and python_version < "4" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
 ruamel.yaml.clib==0.2.2; platform_python_implementation == "CPython" and python_version < "3.8" and python_version >= "3.6"
 ruamel.yaml==0.16.5; python_version >= "3.6" and python_version < "4"
 s3transfer==0.3.4
-schema-salad==7.0.20201119201711; python_version >= "3.6" and python_version < "4"
-setuptools-scm==5.0.1; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
+schema-salad==7.1.20210309094900; python_version >= "3.6" and python_version < "4"
+setuptools-scm==5.0.2; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
 shellescape==3.4.1; python_version >= "3.6" and python_version < "4"
 simplejson==3.17.2; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.6"
 six==1.15.0; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" or python_full_version >= "3.6.0" and python_version >= "3.6" and python_version < "4"
 social-auth-core==3.3.0
 sortedcontainers==2.3.0
 sqlalchemy-migrate==0.13.0
 sqlalchemy-utils==0.36.7
-sqlalchemy==1.3.22; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0")
+sqlalchemy==1.3.23; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0")
 sqlitedict==1.7.0
 sqlparse==0.4.1; python_version >= "3.5"
-starlette==0.13.6; python_version >= "3.6" and python_version < "4.0"
-starlette-context==0.3.1; python_version >= "3.7"
+starlette-context==0.1.3
```

Comment:
@mvdbeek Do we need to add a `>=` pin for starlette-context too or is 0.1.3 fine?

Reply:
There've been many updates between these versions: https://starlette-context.readthedocs.io/en/latest/changelog.html 
Seems risky to use 0.1.3. I'll pin it to `>=0.3.1`

Reply:
https://github.com/galaxyproject/galaxy/pull/11643